server/vulnerabilities/oval/parsed/object_info_state.go

```diff
@@ -0,0 +1,154 @@
+package oval_parsed
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/fleetdm/fleet/v4/server/fleet"
+)
+
+type ObjectInfoState struct {
+	Name           *ObjectStateString      `json:",omitempty"`
```

Comment:
Might be obvious to others, but what does this empty json field name do?

Reply:
It depends on the object associated with the state, for example, in case the object is a dpkg package `Name` refers to the name of the package. It has `omitempty` because is not required.

Reply:
Huh... Does it default to `Name` if the tag itself is not specified? TIL